enhancements/windows-containers/disconnected-support.md

```diff
@@ -0,0 +1,355 @@
+    └── hosts.toml
+
+
+$ cat "$config_path"/registry.access.redhat.com/host.toml
+server = "https://registry.access.redhat.com" # default fallback server since "AllowContactingSource" mirrorSourcePolicy is set
+
+[host."https://example.io/example/ubi-minimal"]
+ capabilities = ["pull"]
+
+[host."https://example.com/example2/ubi-minimal"] # secondary mirror
+ capabilities = ["pull"]
+
+
+$ cat "$config_path"/registry.redhat.io/host.toml
+# "server" omitted since "NeverContactSource" mirrorSourcePolicy is set
+
+[host."https://mirror.example.com"]
+ capabilities = ["pull"]
+
+
+$ cat "$config_path"/docker.io/host.toml
+server = "https://docker.io"
+
+[host."https://docker-mirror.internal"]
+ capabilities = ["pull", "resolve"] # resolve tags
+```
+
+As on the Linux side, it is upon the user to ensure there is no clash between mirrors across their IDMS and ITMS resources.
+
+---
+
+### Test Plan & Infrastructure Needed
+
+In addition to unit testing individual WMCO/WICD packages and controllers, an e2e job will be added to the release repo
+for WMCO's master/release-4.15 branches. A new CI workflow will be created leveraging 
+[existing step-registry steps](https://github.com/openshift/release/tree/master/ci-operator/step-registry/aws/provision/vpc/disconnected),
+which creates a disconnected cluster in AWS or vSphere with hybrid-overlay networking. 
+We'll also need to pre-load the images used in our e2e suite into the cluster's internal registry as part of job setup.
+This workflow will be used to run the existing WMCO e2e test suite to ensure that WMCO runs as expected in disconnected 
+environments with no regression. We will add a few test cases to explicitly check the state of registry settings on Windows
+nodes.
+QE should cover disconnected envrionments on all platforms when validating this feature.
+
+### User Stories
+
+<!--
+Story 1: set proper config_path in containerd_conf.toml
+Story 2: generate hosts.toml(s) at config_path from cluster's IDMS/ITMS resources
+-->
+
+User stories can also be found within the JIRA epic: [WINC-936](https://issues.redhat.com/browse/WINC-936)
+
+### Workflow Description and Variations
+
+**cluster creator** is a human user responsible for deploying a cluster.
+**cluster administrator** is a human user responsible for managing cluster settings including network egress policies.
+
+There are 3 different workflows that affect the disconnected use case.
+1. A cluster creator brings up a new disconnected cluster and configures mirror registry settings at install time
+2. A cluster administrator introduces new mirror registry settings or changes exisiting ones during runtime
+   This would occur through the creation, deletion, or update IDMS/ITMS resources.
+3. A cluster administrator transitions a connected cluster *with existing mirror registry settings* to a disconnected env
+4. A cluster administrator transitions a connected cluster *without existing mirror registry settings* to a disconnected env
```

Comment:
What about transitions from disconnected to connected?

Reply:
should behave the same way as connected to disconnected